### PR TITLE
Add etalab-2.0

### LINKED ISSUE #923: New license request: etalab-2.0
**1.** License Name: Open License 2.0
**2.** Short identifier: ol-2.0
**3.** License Author or steward: Etalab (French mission for open data : https://www.etalab.gouv.fr)
**4.** Comments: The license has not been submitted to OSI because it is mainly targetting open data, not open source.

If modications are required, please let me know.

Thanks!
**5.** Standard License Header: 
**6.** URL: https://www.etalab.gouv.fr/wp-content/uploads/2018/11/open-licence.pdf
**7.** OSI Status: Not Submitted

### PR DESCRIPTION
**Pull request by swinslow**
Fixes #923 

Signed-off-by: Steve Winslow <swinslow@gmail.com>

### CODE REVIEW

**Comment by swinslow** [comment]
@jlovejoy Can you give this a quick once-over, and let me know if you have any concerns? Or go ahead and merge if you'd like. Passed the tests so I'm guessing it is good to go  :)

@bzg Feel free to take a look also -- I know you might not be familiar with SPDX markup, but let me know any questions or concerns.

**Comment by bzg** [comment]
Looks good to me, thanks!

File: src/etalab-2.0.xml [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+      <crossRefs>
+         <crossRef>https://github.com/DISIC/politique-de-contribution-open-source/blob/master/LICENSE.pdf</crossRef>
+         <crossRef>https://raw.githubusercontent.com/DISIC/politique-de-contribution-open-source/master/LICENSE</crossRef>
+      </crossRefs>
+    <notes>English translation can be found here: https://www.etalab.gouv.fr/wp-content/uploads/2018/11/open-licence.pdf</notes>
+    <text>
+
+      <titleText>
+            <p>LICENCE OUVERTE / OPEN LICENCE</p>
+            <optional><p>===================================================================</p></optional>
```
Comment (jlovejoy) [inline src/etalab-2.0.xml]: if the TitleText is optional anyway, do we need the separate optional tags? or at least  just use one? Seems weird to have optional tag inside a TitleText tag in any case??
@swinslow

Comment (swinslow) [reply src/etalab-2.0.xml]: Good question, and I don’t know the right answer... here is a recent one where we included optional tags within titleText: https://github.com/spdx/license-list-XML/blob/master/src/BlueOak-1.0.0.xml

I’m okay with it either way... I am assuming it should be disregarded by matching guidelines either way, so I imagine it wasn’t necessary to add them for the various lines in this one =)

**Pull request merged**